eng/common/scripts/Create-APIReview.ps1

```diff
@@ -56,7 +57,12 @@ function Submit-APIReview($packagename, $filePath, $uri, $apiKey, $apiLabel)
 $packages = @{}
 if ($FindArtifactForApiReviewFn -and (Test-Path "Function:$FindArtifactForApiReviewFn"))
 {
-    $packages = &$FindArtifactForApiReviewFn $ArtifactPath $PackageName
+    $artifactLoc = $ArtifactPath
```

Comment:
Why create a new parameters? Why not adjust the java pipeline to just pass the correct path in as ArtifactPath?

Reply:
I have made this change to pass root artifact directory only in artifact path since that's where all config json files are created before version is updated to alpha. Script needs to know config file location to parse them and find version. Other option is to create those json files in individual artifact locations based on group name and package name directories in the same way build is creating packages. But I thought It's probably simpler if we use additional param to generalize this for all languages and also If build step in any language clears target package directory it will be an issue too if config file is created in it(I am not sure if any lang does it though).

Reply:
I have made the change as discussed to pass config path in new param.